### PR TITLE
Remove rustc-serialize dependency.

### LINKED ISSUE #659: Drop rustc-serialize dependency
It has been deprecated: [announcement](https://users.rust-lang.org/t/deprecation-of-rustc-serialize/10485?u=dtolnay).

Is there any functionality you would require from another library before this would be possible?

### PR DESCRIPTION
**Pull request by emilio**
Fixes #659

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -6,7 +6,6 @@ extern crate env_logger;
 extern crate log;
 extern crate clang_sys;
 extern crate clap;
-extern crate rustc_serialize;
```
Comment (fitzgen) [inline src/main.rs]: lol rly? that's it?

Comment (emilio) [inline src/main.rs]: It is! :)

**Pull request merged**